src/resource/PrivateResourceRoute.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ReactNode, useEffect, useState } from 'react';
```

Comment:
Would it be good to have some unit tests for this PrivateResourceRoute component?

Reply:
I will add 👍